services/motion/builtin/move_attempt.go

```diff
@@ -24,6 +26,8 @@ type moveAttempt struct {
 
 	request      *moveRequest
 	responseChan chan moveResponse
+
+	position, obstacle *replanner
```

Comment:
The rule of zero, one, infinity says this should be a `[]*replanner`, though two is the number where we could be flexible on that. Could compromise for a comment saying "make this a `[]*replanner` if we add any more replanners"

Reply:
I initially had written something like this but its really difficult to do a select statement on a variable list of channel so I punted on that.  I'll add the comment though